src/sdk/analytics/platform/vtex/search.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * More info at: https://www.notion.so/vtexhandbook/Event-API-Documentation-48eee26730cf4d7f80f8fd7262231f84
+ */
+
+import config from 'src/../store.config'
+
+const THIRTY_MINUTES_S = 30 * 60
+const ONE_YEAR_S = 365 * 24 * 3600
+
+const randomUUID = () =>
+  typeof crypto.randomUUID === 'function'
+    ? crypto.randomUUID()
+    : (Math.random() * 1e6).toFixed(0)
+
+const storage = (key: string, expiresSecond: number) => {
```

Comment:
Rename this storage name to a better one, this function is creating a new storage handler for a specific key, wdyt  "storageFactory" or "storageCreator"?